src/main/scala/tech/cryptonomic/conseil/tezos/ApiOperations.scala

```diff
@@ -403,199 +416,65 @@ object ApiOperations {
-              }
-
-            case Some(x) if x == "fitness" =>
-              order.map(_.toLowerCase) match {
-                case Some(x) if x == "asc" => BlocksAction(blockAction.sortBy(_._6.asc))
-                case Some(x) if x == "desc" => BlocksAction(blockAction.sortBy(_._6.desc))
-                case None => BlocksAction(blockAction.sortBy(_._6.desc))
-              }
-            case Some(x) if x == "context" =>
-              order.map(_.toLowerCase) match {
-                case Some(x) if x == "asc" => BlocksAction(blockAction.sortBy(_._7.asc))
-                case Some(x) if x == "desc" => BlocksAction(blockAction.sortBy(_._7.desc))
-                case None => BlocksAction(blockAction.sortBy(_._7.desc))
-              }
-            case Some(x) if x == "signature" =>
-              order.map(_.toLowerCase) match {
-                case Some(x) if x == "asc" => BlocksAction(blockAction.sortBy(_._8.asc))
-                case Some(x) if x == "desc" => BlocksAction(blockAction.sortBy(_._8.desc))
-                case None => BlocksAction(blockAction.sortBy(_._8.desc))
-              }
-            case Some(x) if x == "protocol" =>
-              order.map(_.toLowerCase) match {
-                case Some(x) if x == "asc" => BlocksAction(blockAction.sortBy(_._9.asc))
-                case Some(x) if x == "desc" => BlocksAction(blockAction.sortBy(_._9.desc))
-                case None => BlocksAction(blockAction.sortBy(_._9.desc))
-              }
-            case Some(x) if x == "chain_id" =>
-              order.map(_.toLowerCase) match {
-                case Some(x) if x == "asc" => BlocksAction(blockAction.sortBy(_._10.asc))
-                case Some(x) if x == "desc" => BlocksAction(blockAction.sortBy(_._10.desc))
-                case None => BlocksAction(blockAction.sortBy(_._10.desc))
-              }
-            case Some(x) if x == "hash" =>
-              order.map(_.toLowerCase) match {
-                case Some(x) if x == "asc" => BlocksAction(blockAction.sortBy(_._11.asc))
-                case Some(x) if x == "desc" => BlocksAction(blockAction.sortBy(_._11.desc))
-                case None => BlocksAction(blockAction.sortBy(_._11.desc))
-              }
-            case Some(x) if x == "operations_hash" =>
-              order.map(_.toLowerCase) match {
-                case Some(x) if x == "asc" => BlocksAction(blockAction.sortBy(_._12.asc))
-                case Some(x) if x == "desc" => BlocksAction(blockAction.sortBy(_._12.desc))
-                case None => BlocksAction(blockAction.sortBy(_._12.desc))
-              }
-            case None =>
-              order.map(_.toLowerCase) match {
-                case Some(x) if x == "asc" => BlocksAction(blockAction.sortBy(_._1.asc))
-                case Some(x) if x == "desc" => BlocksAction(blockAction.sortBy(_._1.desc))
-                case None => BlocksAction(blockAction.sortBy(_._1.desc))
-              }
+        case blocks: BlocksAction =>
```

Comment:
Have you done sanity tests on a few fields in postman for sorting?

Reply:
I'm doing now, and In fact I think I've found some problem that I'm looking into to solve.